synapse/lib/stormlib/gen.py

```diff
@@ -109,6 +109,24 @@ class LibGen(s_stormtypes.Lib):
     _storm_lib_path = ('gen',)
 
     _storm_query = '''
+        function orgIdType(name) {
+            ou:id:type:name=$name
+            return($node)
+
+            [ ou:id:type=(gen, name, $name) :name=$name ]
+            return($node)
+        }
+
+        function orgIdNumber(type, value) {
+            $idtype = $orgIdType($type)
+
+            ou:id:number:type=$idtype +:value=$value
```

Comment:
You can lift this by `ou:id:number=($idtype, $value)` rather than the lift+filter